app/lib/efile/line_data.yml

```diff
@@ -169,6 +171,28 @@ IT201_LINE_61:
  voluntary contributions (add lines 46, 58, 59, and 60) .'
 IT201_LINE_62:
   label: '62 Enter amount from line 61 '
+IT201_LINE_67:
+  label: '67 Real property tax credit'
+IT201_LINE_69:
+  label: '69 NYC school tax credit (fixed amount) ('
+IT201_LINE_69A:
+  label: 'NYC school tax credit (rate reduction amount)'
+IT201_LINE_70:
+  label: '70 NYC earned income credit'
+IT201_LINE_72:
+  label: '72 Total New York State tax withheld'
+IT201_LINE_73:
+  label: '73 Total New York City tax withheld'
+IT201_LINE_76:
+  label: '76 Total payments (add lines 63 through 75)'
+IT201_LINE_77:
+  label: '77 Amount overpaid (if line 76 is more than line 62, subtract line 62 from line 76) .'
+IT201_LINE_78:
+  label: '78 Amount of line 77 available for refund (subtract line 79 from line 77)'
+IT201_LINE_78B:
+  label: '78b Total refund after NYS 529 account deposit (subtract line 78a from line 78)'
+IT201_LINE_80:
+  label: '80 Amount you owe (if line 76 is less than line 62, subtract line 76 from line 62)'
 IT213_LINE_3:
```

Comment:
i just re-ordered these so all the IT201 lines are together cuz it was annoying me in debug output